diff --git a/tools/emhermesc/CMakeLists.txt b/tools/emhermesc/CMakeLists.txt--- a/tools/emhermesc/CMakeLists.txt
+++ b/tools/emhermesc/CMakeLists.txt
@@ -12,14 +12,11 @@
   )
 
 if (EMSCRIPTEN)
-  target_link_options(emhermesc PRIVATE "SHELL: -s NODERAWFS=0 -s WASM=1")
   target_link_options(emhermesc PRIVATE "SHELL: -s ALLOW_MEMORY_GROWTH=1")
   target_link_options(emhermesc PRIVATE
     "SHELL: -s EXTRA_EXPORTED_RUNTIME_METHODS=[cwrap,ccall]")
   target_link_options(emhermesc PRIVATE "SHELL: -s MODULARIZE=1")
   target_link_options(emhermesc PRIVATE "SHELL: -s EXPORT_NAME=createHermesc")
   target_link_options(emhermesc PRIVATE
-    "SHELL: -s BINARYEN_ASYNC_COMPILATION=0 -s SINGLE_FILE=1 -s WASM_ASYNC_COMPILATION=0")
-  target_link_options(emhermesc PRIVATE
     "SHELL: -s NODEJS_CATCH_EXIT=0 -s NODEJS_CATCH_REJECTION=0")
 endif ()
